transbank/common/integration_type_host_helper.py

```diff
@@ -0,0 +1,16 @@
+from transbank.common.integration_type import IntegrationType
+
+
+class IntegrationTypeHostHelper(object):
+    @classmethod
+    def get_webpay_host(cls, integration_type: IntegrationType) -> str:
```

Comment:
It's uncommon in python to prefix methods/functions with `set_` or `get_` prefixes. Take a look at other libraries and you will see sometimes they just remove the `get_` part (this would be `webpay_host`) and other times they are written as properties.

Reply:
@leosoto I have made refactor in order to avoid the `get_` in all the code I have written by now.